### DISCUSSION TITLE
Capability to configure local directory

### DISCUSSION THREAD

**Discussion by fbodmer** [microsoft/vscode-azureAutomation#3] [Ideas]
Artifacts on Windows are locally stored in C:\Users\<username>\vscodeAutomation\. Would be nice if this could be changed to any other location. This would help to place the local artifacts under Git Control or backup.

[a comment by an automated account is not shown]

**Reply by JonAmburn** [reply]
Thanks for this info!! 👍 
Can this be a setting in the extension? Or at least documented somewhere? It was very hard to find this :) 
Also, I'm finding it's resetting each time I restart VSCode. Am I missing some way to make this a permanent change? 


By the way - this is really the only problem I've had with this. Thanks for the great work!

**Comment by fbodmer** [comment]
If I change the directory, the vscodeAutomation is retrieved from that directory. However, it seems that AzureAutomationAuthoringToolkitConfig.json is still retrieved from the user directory. This makes working with multiple Automation Accounts a bit difficult as I have to manually update AzureAutomationAuthoringToolkitConfig.json to point to the json files in the other directory.

If the default path in AzureAutomationAuthoringToolkitConfig.json would point to the local directory of that file would be even better. It currently seems to point to the directory of the AzureAutomationAuthoringToolkit.

[a comment by an automated account is not shown]

**Reply by fbodmer** [reply]
Thanks so much, works like a charm!!!
